tests/GameVersionTests.cpp

```diff
@@ -267,4 +267,19 @@ TEST(GameVersion_Tests, LessThanOrEqualsFromEquals)
 	const GameVersion versionTwo(1, 2, 3, 4);
 
 	ASSERT_LE(versionTwo, version);
+}
+
+TEST(GameVersion_Tests, VersionEqual)
```

Comment:
This could use a name that explains more. It should capture the fact that a missing fourth part is the same as one explicitly set to zero, so that it's distinguished from EqualityCanBeTrue().

Reply:
Renamed to `GameVersionEqualMissingFourthPartIsSameAsZero`.